packages/core/types/zapier.generated.d.ts

```diff
@@ -185,7 +185,7 @@ export type RefResource = string;
  */
 export type FieldChoices =
   | { [k: string]: unknown }
```

Comment:
I know this line is out of scope. Just thinking out loud here. It can be written as `Record<string, unknown>` alternatively.